engine/language-server/src/main/scala/org/enso/languageserver/text/CollaborativeBuffer.scala

```diff
@@ -480,27 +592,35 @@ object CollaborativeBuffer {
 
   case object IOTimeout
 
+  private case class AutoSave(
+    clientId: ClientId,
+    clientVersion: ContentVersion
+  )
+
   /** Creates a configuration object used to create a [[CollaborativeBuffer]]
     *
     * @param bufferPath a path to a file
     * @param fileManager a file manager actor
     * @param runtimeConnector a gateway to the runtime
     * @param timeout a request timeout
     * @param versionCalculator a content based version calculator
+    * @param timingsConfig a config with timing/delay values
     * @return a configuration object
     */
   def props(
     bufferPath: Path,
     fileManager: ActorRef,
     runtimeConnector: ActorRef,
-    timeout: FiniteDuration = 10 seconds
+    timeout: FiniteDuration = 10 seconds,
+    timingsConfig: TimingsConfig
```

Comment:
Isn't `timeout` also a `timingsConfig`? E.g. shouldn't that be somehow merged?

Reply:
Yeah, I was wondering if I should push it in this PR. I will do it.